src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/ChannelSubType.java

```diff
@@ -0,0 +1,14 @@
+package de.tum.in.www1.artemis.domain.metis.conversation;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum ChannelSubType {
+    @JsonProperty("general")
```

Comment:
I don't see the JsonProperties in other enums. Do we actually need them here?

Reply:
Yes, since we use the enum in the ChannelDTO, deserialization of lowercase strings would not be correctly mapped to the corresponding enum value.